include/WCSimWCPMT.hh

```diff
@@ -16,18 +16,21 @@ class WCSimWCPMT : public G4VDigitizerModule
 {
 public:
   
-  WCSimWCPMT(G4String name, WCSimDetectorConstruction*);
+  WCSimWCPMT(G4String name, WCSimDetectorConstruction*, G4String detectorElement);
   ~WCSimWCPMT();
   
    void ReInitialize() { DigiHitMapPMT.clear(); TriggerTimes.clear(); }
-    
+   // Set Relative Digitized Hit time mode and reinitialize first_time hit
+   void SetRelativeDigitizedHitTime (bool val) { RelativeHitTime = val; fFirst_Time = -1; }
```

Comment:
If you check the code WCSimWCPMT.cc, there is a check that fFirst_time = -1 to identify if we are in relative time or not. So, moving to your proposal would need a change in the code. I personally think that the code is fine as it is, but I am open to modify  it if needed. Please let me know if you see any need of changing it, and any potential application.

Reply:
Thank you for the explanation. I understand the need.

Reply:
Sorry, I was missing this message, actually, @akutsuR proposal is good since we have:

`   void SetRelativeDigitizedHitTime (bool val) { RelativeHitTime = val; fFirst_Time = -1; }
`
`   void SetRelativeDigitizedHitTime (bool val, G4double time) { RelativeHitTime = val; fFirst_Time = time; }
`
These fFirst_Time = -1 is used for 20" PMT, whereas fFirst_Time = time is used for mPMT (feeding them with the first time of the 20" PMT hits)